### PR TITLE
Bmp 666 align json editor with original

### PR DESCRIPTION
**Pull request by Danielecina**

### CODE REVIEW

**Comment by fredmaggiowski** [comment]
i'd update the repo with the original repo in another PR, not this one...

There are now 58 files changed and it's pretty hard to know which one are your commits

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,11 +1,29 @@
 ### Unreleased
+
 - added option `max_depth` used to specify the maximum depth of level's schema that have to be rendered
```
Comment (dibericky) [inline CHANGELOG.md:3]: This changes in which version has been released?
I think that CHANGELOG.md has been merged wrongly

Comment (Danielecina) [reply CHANGELOG.md:3]: Compared to master I did not modify the changelog except by putting down the minor ones made in the original repo

**Comment by Danielecina** [comment]
> i'd update the repo with the original repo in another PR, not this one...
> There are now 58 files changed and it's pretty hard to know which one are your commits

I didn't commit anything in this branch. I only aligned json-editor

**Comment by fredmaggiowski** [comment]
> > i'd update the repo with the original repo in another PR, not this one...
> > There are now 58 files changed and it's pretty hard to know which one are your commits
> 
> I didn't commit anything in this branch. I only aligned json-editor

my bad, I gave for granted this was already the new feature 😅

**Comment by fredmaggiowski** [comment]
LGTM

**Review by fredmaggiowski — APPROVED** [review]

**Review by davidebianchi — APPROVED** [review]

**Pull request merged**